bigflow/configuration.py

```diff
@@ -111,3 +114,16 @@ def _resolve_placeholders(self, value, variables: dict):
         else:
             return value
 
+
+class DeploymentConfig(Config):
+    def __init__(self,
+                 name: str,
+                 properties: dict,
+                 is_master: bool = True,
+                 is_default: bool = True,
+                 environment_variables_prefix: str = DEFAULT_CONFIG_ENV_VAR_PREFIX):
```

Comment:
nit: consts are not actual consts in PY, so someone might change variable 'DEFAULT_CONFIG_ENV_VAR_PREFIX' and will not be reflected on this class/parameter. Maybe `None` OR drop the 'const variable'.

Reply:
done